### PR TITLE
content updated & breadcrumb added

### PR DESCRIPTION
**Pull request by whoafridi**
mishar bhai check, updated & add the code into main branch

### CODE REVIEW

**Comment by sheikhmishar** [comment]
Sure, let me review and merge it

**Review by sheikhmishar — COMMENTED** [review]
Can you please explain this line?

```html
<div style="--bs-breadcrumb-divider: '>'; color: #fff;" aria-label="breadcrumb">
```

File: packages/character-packages/magician-package/index.html [hunk 1]
```diff
@@ -218,16 +224,16 @@ <h1 class="page-title"><br> Packages</h1>
                     <div class="tab-content" id="pills-tabContent">
                         <div class="tab-pane fade" id="pills-home" role="tabpanel" aria-labelledby="pills-home-tab">
                             <ul>
-                                <li><a href="../../party-packages/face-painting/">Face Painting & Temporary Tattoo</a></li>
-                                <li><a href="../../party-packages/balloon-sculpting/">Balloon Sculpting</a></li>
+                                <li><a href="../../party-packages/face-painting-hire/">Face Painting & Temporary Tattoo</a></li>
```
Comment (sheikhmishar) [inline packages/character-packages/magician-package/index.html]: What was the instruction provided by the SEO? Only to change the URL? Or change both the URL and the InnerText?

```html
<a href="../../party-packages/face-painting-hire/">Face Painting & Temporary Tattoo</a>
```

File: packages/character-packages/magician-package/index.html [hunk 2]
```diff
@@ -364,7 +368,7 @@ <h3 style="text-align: right; font-size: 18px;">1.5 hours for
                 </a>
             </div>
             <div class="col-md-3">
-                <a href="../fairy-package/">
+                <a href="../fairy-party-hire/">
                     <div class="img-card">
                         <img src="../../../img/fairy.png" alt="kidsdream-fairy-Package-img">
                         <div class="img-title">Fairy Package</div>
```
Comment (sheikhmishar) [inline packages/character-packages/magician-package/index.html]: Shouldn't it be 'Fairy Party Hire'?

```html
<div class="img-title">Fairy Package</div>
```

**Comment by whoafridi** [comment]
@sheikhmishar the code is for breadcrumb copied from bootstrap docs.
Screenshot from 2023-06-13 18-31-08

File: css/style.css [hunk 3]
```diff
@@ -349,4 +349,9 @@ nav.navbar .navbar-nav .nav-item .btn-group a:hover {
 } */
 .breadcrumb-item + .breadcrumb-item::before {
 	color: #fff;
-}
\ No newline at end of file
+}
+
+.breadcrumb-style{
```
Comment (sheikhmishar) [inline css/style.css]: rename the class to .breadcrumb-divider-angle
remove the color property. color has nothing to do with breadcrumb divider
text-white class is already there for this

**Comment by whoafridi** [comment]
> Can you please explain this line?
> 
> ```
> <div style="--bs-breadcrumb-divider: '>'; color: #fff;" aria-label="breadcrumb">
> ```

this is a bootstrap class to add breadcrumb in website

**Comment by whoafridi** [comment]
> Can you please explain this line?
> 
> ```
> <div style="--bs-breadcrumb-divider: '>'; color: #fff;" aria-label="breadcrumb">
> ```

**Comment by sheikhmishar** [comment]
What was the instruction provided by the SEO? Only to change the URL? Or change both the URL and the InnerText?
```html
<a href="../../party-packages/face-painting-hire/">Face Painting & Temporary Tattoo</a>
```

**Comment by sheikhmishar** [comment]
Shouldn't it be 'Fairy Party Hire'?

<div class="img-title">Fairy Package</div>

**Comment by sheikhmishar** [comment]
rename the class to .breadcrumb-divider-angle
remove the color property. color has nothing to do with breadcrumb divider
text-white class is already there for this

**Review by sheikhmishar — APPROVED** [review]
Good job, Afridi. Merging the code with the `main` branch

**Pull request merged**